constraintlayout/core/src/main/java/androidx/constraintlayout/core/state/Transition.java

```diff
@@ -833,7 +834,8 @@ private WidgetState getWidgetState(String widgetId,
         if (widgetState == null) {
             widgetState = new WidgetState();
             mBundle.applyDelta(widgetState.mMotionControl);
-
+            widgetState.mMotionWidgetStart.updateMotion( widgetState.mMotionWidgetStart);
```

Comment:
Can you double check this? Seems unnecessary

Reply:
removed